bool changeLane
a3_link[] links
int16 src
int16 dst
bool isStop
